TheMapCloudWebServices/osma_plugin.cfg

```diff
@@ -0,0 +1,4 @@
+[plugin_configuration]
+name = TheMapCloudWebServices
+title = OSMA Web Services
+help_url = http://wms.locationcentre.co.uk/wiki/index.php/OSMA_Web_Services
```

Comment:
Clicking on the WIKI button caused the API docs to be shown rather than the wiki!